ui/component/or-attribute-card/src/index.ts

```diff
@@ -744,11 +744,11 @@ export class OrAttributeCard extends LitElement {
             assetId,
             attributeName,
             {
-                type: "lttb",
+                type: "all",
                 fromTimestamp: this._startOfPeriod,
                 toTimestamp: this._endOfPeriod,
                 amountOfPoints: 20,
-            } as AssetDatapointLTTBQuery
+            } as AssetDatapointAllQuery
```

Comment:
Is this intentional?

Reply:
To additionally comment on this; we're intentionally using `LTTB` in the `or-attribute-card` here.

Reply:
Thanks but is it a change that makes sense for everything or just this fleet management application?